src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+function toNumber(string) {
+  const num = +string.slice(1).split(',').join('');
+
+  return num;
+}
+
+const headings = document.querySelector('thead');
+const table = document.querySelector('tbody');
+
+headings.addEventListener('click', (e) => {
+  const criterion = e.target.cellIndex;
+  const sortedList
+   = [...table.children].sort((a, b) => {
+     let current = a.cells[criterion].innerText;
+     let next = b.cells[criterion].innerText;
+
+     if (criterion === 3 || criterion === 4) {
```

Comment:
better move 3 and 4 to consts with clear names, so that it's clear what those numbers mean. Example:
```suggestion
     const ageCellIndex = 3
     const salaryCellIndex = 4
     
     if (criterion === ageCellIndex || criterion === salaryCellIndex) {
```